jupyterhub/files/hub/cull_idle_servers.py

```diff
@@ -168,16 +176,24 @@ def handle_server(user, server_name, server):
                 log_name, format_td(age), format_td(inactive))
             return False
 
+        body = None
         if server_name:
             # culling a named server
-            delete_url = url + "/users/%s/servers/%s"%(
-                quote(user['name']), quote(server['name'])
+            delete_url = url + "/users/%s/servers/%s" % (
+                quote(user['name']),
+                quote(server['name']),
             )
+            if remove_named_servers:
+                body = json.dumps({"remove": True})
         else:
             delete_url = url + '/users/%s/server' % quote(user['name'])
 
         req = HTTPRequest(
-            url=delete_url, method='DELETE', headers=auth_header,
+            url=delete_url,
+            method='DELETE',
+            headers=auth_header,
+            body=body,
+            allow_nonstandard_methods=True,
```

Comment:
`method='DELETE'` is unchanged, why is `allow_nonstandard_methods=True` now required?

Reply:
Because we (sometimes) pass a `body` which isn't allowed by the strict mode.